### PR TITLE
Retail use productname #12185 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

For product names make use of macros.  the 'susemgr' does not make sense in the uyuni context.
Done for the first sections. I'll start a new PR for the rest.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links
It partially addresses https://github.com/SUSE/spacewalk/issues/12185

### CODE REVIEW

**Comment by keichwa** [comment]
I especially would like to know whether the redefinition of the 'smr' macro in entities.adoc with 'productname' would work?

File: modules/retail/pages/retail-requirements.adoc [hunk 1]
```diff
@@ -63,14 +67,33 @@ xref:installation:general-requirements.adoc[].
...
+// Shall we also mention Leap?
+For more information, see the documentation of the underlying system (in this case: {sles}{nbsp}15).
+
+For more information on {smr} POS terminals, see documentation on {productname} Salt clients (xref:client-configuration:client-config-overview.adoc[]).
+
+
+
+=== UEFI Secure Booting Requirements
+
+// Shall we also mention Leap?
```
Comment (aaannz) [inline modules/retail/pages/retail-requirements.adoc:96]: How binding word `supported` is here in docs? For SLE it is easy, as we provide support for SLE. But for Leap and Uyuni it is different picture.
UEFI Secure Boot should work on Leap, but I'm not sure about the supported statement.

**Comment by keichwa** [comment]
Ondřej Holeček <notifications@github.com> writes:

> How binding word supported is here in docs? For SLE it is easy, as we
> provide support for SLE. But for Leap and Uyuni it is different
> picture.

Nice spot, a can of worms.  I'll probably ifdef'ing it.

> UEFI Secure Boot should work on Leap, but I'm not sure about the
> supported statement.

Thanks for feedback!

-- 
Karl Eichwalder

**Comment by keichwa** [comment]
Done.  I also changed the support statement with the x86_64 architecture.  For uyuni I now say, it is tested on that arch.  @aaannz if we should say something better please provide feedback!

@Loquacity what do you think?

File: modules/retail/pages/retail-network-arch.adoc [hunk 2]
```diff
@@ -16,8 +16,16 @@ We recommend you have a fast network connection between the branch server and it
...
+
+ifeval::[{suma-content} == true]
+{smr} Branch Servers are implemented as enhanced {productname} Proxies.
+For technical background information on {smr} Branch Servers, see also the documentation on {productname} Proxy servers (xref:installation:install-proxy-unified.adoc[]).
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+{smr} Branch Servers are implemented as enhanced {productname} Proxies.
+For technical background information on {smr} Branch Servers, see also the documentation on {productname} Proxy servers (xref:installation:install-proxy-uyuni.adoc[]).
+endif::[]
```
Comment (Loquacity) [inline modules/retail/pages/retail-network-arch.adoc:28]: Can we do these without the parenths? Use the standard "for more ..., see..."

File: modules/retail/pages/retail-requirements.adoc [hunk 3]
```diff
@@ -3,19 +3,29 @@
...
 [IMPORTANT]
 ====
 {smr} is only supported on the {x86_64} architecture.
 ====
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+[IMPORTANT]
+====
+{smr} is only tested on the {x86_64} architecture.
```
Comment (Loquacity) [inline modules/retail/pages/retail-requirements.adoc:19]:
```suggestion
{smr} is tested on {x86_64} architecture.
```

File: modules/retail/pages/retail-requirements.adoc [hunk 4]
```diff
@@ -63,9 +77,9 @@ xref:installation:general-requirements.adoc[].
 
 == Network Requirements
 
-* The {susemgr} Server requires a reliable and fast WAN connection.
-* The branch server requires a reliable WAN connection, to reach the {susemgr} Server.
-* If you are using a dedicated network, the branch server requires at least two network interfaces: one connected to the WAN with reachable {susemgr} server, and one connected to the internal branch LAN.
+* The {productname} Server requires a reliable and fast WAN connection.
+* The branch server requires a reliable WAN connection, to reach the {productname} Server.
+* If you are using a dedicated network, the branch server requires at least two network interfaces: one connected to the WAN with reachable {productname} server, and one connected to the internal branch LAN.
```
Comment (Loquacity) [inline modules/retail/pages/retail-requirements.adoc:82]:
```suggestion
* If you are using a dedicated network, the branch server requires at least two network interfaces: one connected to the WAN with a reachable {productname} Server, and one connected to the internal branch LAN.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/retail/pages/retail-network-arch.adoc (hunk 2 above)
Comment (keichwa) [reply modules/retail/pages/retail-network-arch.adoc:28]: done. we will have to change other retail files accordingly :)

**Review by Loquacity — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Comment by Loquacity** [comment]
> I especially would like to know whether the redefinition of the 'smr' macro in entities.adoc with 'productname' would work?

Just noticed this question. I think we would need to define {smr} as "for Retail" and use it as {productname} {smr}.

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Just noticed this question. I think we would need to define {smr} as "for Retail" and use it as {productname} {smr}.

Yes, this would work.  Thanks!

-- 
Karl Eichwalder

**Review by Loquacity — APPROVED** [review]
Did you check the built docs?

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
Yep, let's get it in before 4.1.2 :+1:

**Comment by keichwa** [comment]
> Yep, let's get it in before 4.1.2 +1

Thus far, I did not intent to back-port it to 4.1, but it would make sense.

**Comment by Loquacity** [comment]
> > Yep, let's get it in before 4.1.2 +1
> 
> Thus far, I did not intent to back-port it to 4.1, but it would make sense.

I haven't cut the branch yet. If you want it in, you will need to merge it today.

**Comment by keichwa** [comment]
> > > Yep, let's get it in before 4.1.2 +1
> > 
> > 
> > Thus far, I did not intent to back-port it to 4.1, but it would make sense.
> 
> I haven't cut the branch yet. If you want it in, you will need to merge it today.

For 4.1, I  had to create a new PR: https://github.com/uyuni-project/uyuni-docs/pull/533
cherry-pick did not work.

**Pull request merged**